src/allotropy/parsers/biorad_bioplex_manager/biorad_bioplex_manager_parser.py

```diff
@@ -114,7 +113,9 @@ def _get_model(self, data: Et.Element, filename: str) -> Model:
             multi_analyte_profiling_aggregate_document=MultiAnalyteProfilingAggregateDocument(
                 device_system_document=device_document,
                 data_system_document=self._get_data_system_document(
```

Comment:
I know this falls outside the scope of this refactor, but it seems unnecessary to have a method that just returns an instance of `DataSystemDocument` with the arguments passed.

Would you mind removing this code redundancy?